contracts/LoanManager.sol

```diff
@@ -22,13 +22,13 @@ contract LoanManager is Restricted, TokenReceiver {
     enum LoanState { Open, Repaid, Defaulted, Collected } // NB: Defaulted state is not stored, only getters calculate
 
     struct LoanProduct {
-        uint minDisbursedAmount; // 0: with decimals set in AugmintToken.decimals
-        uint32 term;            // 1
-        uint32 discountRate;    // 2: discountRate in parts per million , ie. 10,000 = 1%
-        uint32 collateralRatio; // 3: loan token amount / colleteral pegged ccy value
-                                //      in parts per million , ie. 10,000 = 1%
-        uint32 defaultingFeePt; // 4: % of collateral in parts per million , ie. 50,000 = 5%
-        bool isActive;          // 5
+        uint minDisbursedAmount;    // 0: minimum loanAmount, with decimals set in AugmintToken.decimals (i.e. token amount)
+        uint32 term;                // 1: term length (in seconds)
+        uint32 discountRate;        // 2: discountRate (in parts per million, i.e. 10,000 = 1%)
+        uint32 collateralRatio;     // 3: repayment (token amount) / collateral (token amount), (in parts per million, i.e. 10,000 = 1%)
```

Comment:
assuming we will have the option to originate a loan with more collateral (or add extra collateral later) then we should rename this to `minCollateralRatio`

Reply:
that's the initial value, not the minimum. (also since its an inverse it would technically be a maximum)
we've already renamed the actual minimum (~margin) to minCollateralRatio